### ISSUE TITLE
Missing documentation for compareStrings

### ISSUE THREAD

**Issue by gevro** [Bioconductor/Biostrings#80]
Hi,
Documentation of compareStrings is missing an explanation for what it does with input of:
pattern="PairwiseAlignments", subject="missing"

I believe the best place to document this would be:
PairwiseAlignments-class {Biostrings}

**Comment by hpages** [comment]
This is clarified in **Biostrings** 2.65.4. The man page for `compareStrings()` (`?compareStrings`) now says:
```
pattern, subject: The strings to compare. Can be of type ‘character’,
          ‘XString’, ‘XStringSet’, ‘AlignedXStringSet’, or, in the case
          of ‘pattern’, ‘PairwiseAlignments’.

          If the first argument of ‘compareStrings()’ (‘pattern’) is a
          ‘PairwiseAlignments’ object, then the second argument
          (‘subject’) must be missing. In this case ‘compareStrings(x)’
          is equivalent to ‘compareStrings(pattern(x), subject(x))’.
```

**Biostrings** 2.65.4 should become available in BioC 3.16 (devel) in the next 24-48 hours via `BiocManager::install()`.

**Comment by gevro** [comment]
Thank you
